io_mesh_w3d/__init__.py

```diff
@@ -207,10 +206,10 @@ def error(self, msg):
     def execute(self, context):
         if self.filepath.lower().endswith('.w3d'):
             from .w3d.import_w3d import load
-            load(self, import_settings={})
+            load(self, {})
```

Comment:
I don't think this is particular well readable. When we have no import_settings yet you could remove that parameter aswell